test/e2e/utils/consts.go

```diff
@@ -0,0 +1,14 @@
+package utils
```

Comment:
I like the idea but `utils` is an anti-pattern. Let's think of a better alternative.

Reply:
How about ```testconsts```?

Reply:
@fromanirh ^

Reply:
on second thought the arguments with prevail is avoid kitchen-sink packages.
`testconsts` is a tiny improvement, but moving all the constants into a separate package is a sidegrade after all.
I think the best course of action is to repurpose this PR to add the 
```
// TestNodeLabel unique label for the node that
	TestNodeLabel = "rte-e2e-test-node"
```
constant, and leave the rest of the constants in the `nodes` package. All the constants belong to the node area after all, and overall keeping them there is the lesser evil and a cleanish solution.

Reply:
Done